cloud-azure/src/main/java/com/sequenceiq/cloudbreak/cloud/azure/client/AzureClientService.java

```diff
@@ -23,6 +25,9 @@
     @Inject
     private AuthenticationContextProvider authenticationContextProvider;
 
+    @Inject
+    private Tracer tracer;
```

Comment:
I think it would be nice if you would inject a new class here, like `AzureTracer` and it would have the `public Interceptor getTracingInterceptor()` method and you could pass that along. So it would have the responsibility creating the interceptor instead of `AzureClientCredentials`. What do you think?

Reply:
Im going to create TracingInterceptor as a separate class and inject it directly instead